executor/aggregate_test.go

```diff
@@ -121,8 +121,8 @@ func (s *testSuite1) TestAggregation(c *C) {
 	result.Check(testkit.Rows())
 	result = tk.MustQuery("select count(*) from t a join t b where a.c < 0")
 	result.Check(testkit.Rows("0"))
-	result = tk.MustQuery("select sum(b.c), count(b.d), a.c from t a left join t b on a.c = b.d group by b.d order by b.d")
-	result.Check(testkit.Rows("<nil> 0 <nil>", "8 12 1", "5 2 3"))
+	result = tk.MustQuery("select sum(b.c), count(b.d), max(a.c) from t a left join t b on a.c = b.d group by b.d order by b.d")
```

Comment:
Why change this test?

Reply:
The result of `first_row(a.c)` is nondeterministic since one group may contain more than one row. Thus I change it to `max(a.c)`